traffic_ops/traffic_ops_golang/crconfig/handler.go

```diff
@@ -271,11 +270,10 @@ func SnapshotOldGUIHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	cdn := inf.Params["cdn"]
-
 	// We never store tm_path, even though low API versions show it in responses.
 	crConfig, err := Make(inf.Tx.Tx, cdn, inf.User.UserName, r.Host, inf.Config.Version, inf.Config.CRConfigUseRequestHost, false)
 	if err != nil {
-		writePerlHTMLErr(w, r, inf.Tx.Tx, errors.New(r.RemoteAddr+" making CRConfig: "+err.Error()), err)
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, err, errors.New(r.RemoteAddr+" making CRConfig: "+err.Error()))
```

Comment:
The method signature for `api.HandleErr` is:
```
func HandleErr(w http.ResponseWriter, r *http.Request, tx *sql.Tx, statusCode int, userErr error, sysErr error)
```
So, we don't want to pass the `err` into the Handler because it will be assumed to be a userErr. Perhaps `api.HandleErr` is a bit too general handling all types of errors. I might argue for a `api.HandleUserErr` and a `api.HandleServerErr`. But in any case, for the time being we want to pass `nil` along instead.
```
api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New(r.RemoteAddr+" making CRConfig: "+err.Error()))
```

Reply:
Yeah makes sense. I was trying to follow what was being done for `writePerlHTMLErr`, where both the user error and the system error were being passed. Changed now.